string/regex matcher/8449789e-3b23-4bf5-9c33-931015049dc5.json

```diff
@@ -1,28 +1,38 @@
 {
-  "description": "Executes a global regex pattern match.\n\nFlavour depends on the running environment.",
+  "description": "Matches a string to a non-global regular expression, and returns substrings that match the regular expression's groups.\n\n",
```

Comment:
Actually it not only returns the groups and only returns them if they are present in the regexp so I would rephrase the description or keep it simple